Microsoft.Toolkit.Uwp.UI.Controls/TabView/TabView.xaml

```diff
@@ -0,0 +1,734 @@
+                              Grid.Column="0"
+                              Content="{TemplateBinding TabStartHeader}"
+                              ContentTemplate="{TemplateBinding TabStartHeaderTemplate}" />
+
+            <ScrollViewer x:Name="ScrollViewer"
+                          Grid.Row="1"
+                          Grid.Column="1"
+                          AutomationProperties.AccessibilityView="Raw"
+                          BringIntoViewOnFocusChange="{TemplateBinding ScrollViewer.BringIntoViewOnFocusChange}"
+                          HorizontalScrollBarVisibility="{TemplateBinding ScrollViewer.HorizontalScrollBarVisibility}"
+                          HorizontalScrollMode="{TemplateBinding ScrollViewer.HorizontalScrollMode}"
+                          IsDeferredScrollingEnabled="{TemplateBinding ScrollViewer.IsDeferredScrollingEnabled}"
+                          IsHorizontalRailEnabled="{TemplateBinding ScrollViewer.IsHorizontalRailEnabled}"
+                          IsHorizontalScrollChainingEnabled="{TemplateBinding ScrollViewer.IsHorizontalScrollChainingEnabled}"
+                          IsVerticalRailEnabled="{TemplateBinding ScrollViewer.IsVerticalRailEnabled}"
+                          IsVerticalScrollChainingEnabled="{TemplateBinding ScrollViewer.IsVerticalScrollChainingEnabled}"
+                          Style="{StaticResource TabViewScrollViewer}"
+                          TabNavigation="{TemplateBinding TabNavigation}"
+                          VerticalScrollBarVisibility="{TemplateBinding ScrollViewer.VerticalScrollBarVisibility}"
+                          VerticalScrollMode="{TemplateBinding ScrollViewer.VerticalScrollMode}"
+                          ZoomMode="{TemplateBinding ScrollViewer.ZoomMode}">
+                <ItemsPresenter x:Name="TabsItemsPresenter"
+                                Padding="{TemplateBinding Padding}" />
+            </ScrollViewer>
+
+            <ContentPresenter Grid.Row="1"
+                              Grid.Column="3"
+                              HorizontalAlignment="Left"
+                              Content="{TemplateBinding TabActionHeader}"
+                              ContentTemplate="{TemplateBinding TabActionHeaderTemplate}" />
+
+            <ContentPresenter Grid.Row="1"
+                              Grid.Column="5"
+                              Content="{TemplateBinding TabEndHeader}"
+                              ContentTemplate="{TemplateBinding TabEndHeaderTemplate}" />
+
+            <ContentPresenter x:Name="TabContentPresenter"
+                              Grid.Row="2"
+                              Grid.ColumnSpan="6"
+                              Background="{TemplateBinding Background}"
+                              BorderBrush="{TemplateBinding BorderBrush}"
+                              BorderThickness="{TemplateBinding BorderThickness}" />
+
+            <ContentPresenter Grid.Row="3"
+                              Grid.ColumnSpan="6"
```

Comment:
Only five columns

Reply:
6 columns is right (they're indexed 0 - 5 in the comments):
```
            <Grid.ColumnDefinitions>
                <!--  0 TabStartHeader  -->
                <ColumnDefinition Width="Auto" />
                <!--  1 Tabs  -->
                <ColumnDefinition Width="Auto"
                                  local:TabView.ConstrainColumn="True"
                                  local:TabView.IgnoreColumn="True" />
                <!--  2 Reserved: Overflow Drop-down  -->
                <ColumnDefinition Width="Auto" />
                <!--  3 Auto Add Button  -->
                <ColumnDefinition Width="Auto" />
                <!--  4 Padding  -->
                <ColumnDefinition Width="*"
                                  local:TabView.IgnoreColumn="True" />
                <!--  5 TabEndHeader  -->
                <ColumnDefinition Width="Auto" />
            </Grid.ColumnDefinitions>
```